esrally/driver/runner.py

```diff
@@ -301,8 +303,15 @@ def mandatory(params, key, op):
     try:
         return params[key]
     except KeyError:
-        raise exceptions.DataError("Parameter source for operation '%s' did not provide the mandatory parameter '%s'. Please add it to your"
-                                   " parameter source." % (str(op), key))
+        raise exceptions.DataError(
+            f"Parameter source for operation '{str(op)}' did not provide the mandatory parameter '{key}'. "
+            f"Add it to your parameter source and try again.")
+
+
+def remove_prefix(string, prefix):
```

Comment:
Too bad we don't have Python 3.9 as minimum version requirement as this has been recently added to the [standard library](https://docs.python.org/3/library/stdtypes.html#str.removeprefix). I wonder whether we should put a TODO into the code?

Reply:
Yes I noticed that too while coding writing this. Added a TODO in a3cab0e5a5470d7dcdc1964d6c48c8d0a32969ab.